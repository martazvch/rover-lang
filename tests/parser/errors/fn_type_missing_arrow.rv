fn factory(arg: float) -> fn(float, bool) float { }

