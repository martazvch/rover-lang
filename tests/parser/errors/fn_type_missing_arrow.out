expect_arrow_before_fn_type

